cmd/snap/cmd_list_test.go

```diff
@@ -0,0 +1,83 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
```

Comment:
this needs now a build tag:

// +build !integrationcoverage

(there's run-check test about this)